src/app/tests/spec/SpecMoreInfoDialog.js

```diff
@@ -79,12 +79,25 @@ require([
         });
         describe('onSubmitClick', function () {
             var value = 'blah';
+            var part = 'spine';
```

Comment:
I think that a test should be added that asserts that this value is gathered in `getData()`.

Reply:
I will write a test now